### PR TITLE
Add components and design the calculator

### PR DESCRIPTION
**Pull request by iBel251**
- Remove all the boilerplate from CRA (text, images, styles).
- Create a directory called components and Calculator.js file.
- Created a calculator design
image

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    <from>
+      <input type="text" placeholder="0" />
+    </from>
+    <div className="keypad">
+      {keys.map((btn) => (<button type="button" key={btn.name} className={btn.class}>{btn.name}</button>))}
+    </div>
+  </div>
+);
+
+export default Calculator;
```
Comment (Wes-Isaac) [inline src/components/Calculator.js]: - [ ] Great job so far, just like you did in `App.js` kindly use class-based components, as it's one of the requirements.

image

You got this 👍.

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @iBel251, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linters are set up properly ✔️.
- Readme looks professional 👀.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Calculator.js (hunk 1 above)
Comment (rloterh) [reply src/components/Calculator.js]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Imran,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Highlights 👍
- Descriptive Pull Request
- Class-based components properly implemented
- No linter errors


<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**